From 76d11f2a21e5e305d0e3aaf441293e6d16a5f488 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 18 Jan 2023 14:31:05 +0100
Subject: [PATCH 248/954] feat(stm32mp2): add DDR firmware OID

To manage DDR firmware with TRUSTED_BOARD_BOOT, we need a dedicated
HASH OID. It will be used by cert_tool, and added in STM32MP2
cot-descriptor device tree file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a5c14a3b86d07c22ab26da6b1ed554b71d315866 feat(stm32mp2): add DDR firmware OID]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I7442b38e3219c495e2c8496ae9d33c8ef46401cd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/285743
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 plat/st/stm32mp2/include/platform_oid.h | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 plat/st/stm32mp2/include/platform_oid.h

diff --git a/plat/st/stm32mp2/include/platform_oid.h b/plat/st/stm32mp2/include/platform_oid.h
new file mode 100644
index 0000000000..da5b305a97
--- /dev/null
+++ b/plat/st/stm32mp2/include/platform_oid.h
@@ -0,0 +1,7 @@
+/*
+ * Copyright (C) 2023, STMicroelectronics - All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#define DDR_FW_HASH_OID		"1.3.6.1.4.1.4128.2300.1"
-- 
2.53.0

